pkg/descheduler/strategies/types.go

```diff
@@ -0,0 +1,34 @@
+package strategies
+
+import (
+	"context"
+	v1 "k8s.io/api/core/v1"
+	clientset "k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/util/workqueue"
+	"sigs.k8s.io/descheduler/pkg/api"
+	"sigs.k8s.io/descheduler/pkg/descheduler/evictions"
+)
+
+// StrategyFunction defines the function signature for each strategy's main function
+type StrategyFunction func(
```

Comment:
I'd rather keep this type private until we discuss how to refactor the way strategies are initialized.